circle.yml

```diff
@@ -1,15 +1,16 @@
 machine:
   java:  # see <https://github.com/svenkreiss/html5validator#integration-with-circleci>
     version: oraclejdk8
+  environment:
+    NOKOGIRI_USE_SYSTEM_LIBRARIES: true # Faster installation of nokogiri, required by html-proofer
 
 dependencies:
   post:
-    - bundle exec jekyll --version  # print it out
-    - sudo pip install html5validator
-
+    - bundle exec jekyll --version # print it out
 test:
   override:
     - bundle exec jekyll doctor
     - bundle exec jekyll build
-    - grep "css/main.css" _site/index.html  # smoke test that something was actually compiled
+    - bundle exec htmlproofer ./_site --assume-extension --check-html --disable-external --empty-alt-ignore --enforce-https --url-ignore "/labonneboite|modernisation/"
```

Comment:
Mmh. I'm not sure this `enforce-https` + whitelist scales.

Ok to try, but we need to document how to handle such validation errors (i.e. “update the whitelist”) in the CONTRIBUTING.